### DISCUSSION TITLE
I am using Whisper to transcribe and I am getting the below error, unable to figure out the issue.

### DISCUSSION THREAD

**Discussion by Suma-Rajashankar** [openai/whisper#1709] [Q&A]
I am using Whisper to transcribe an audio file. I have installed Python3.9, ffmpeg and the associated dependencies, and openai-whisper==20230308. I could import whisper, but when I try to run transcribe:

`
audio = f'audio/testaudio.wav'
model = whisper.load_model("base") 
`
`result = model.transcribe(audio, fp16=False,language="en")`

...I get the following error:

```
ValueError                                Traceback (most recent call last)
Cell In [32], line 1
----> 1 result = model.transcribe(audio, fp16=False, language="en")

File ~/.local/lib/python3.9/site-packages/torch/autograd/grad_mode.py:27, in _DecoratorContextManager.__call__.<locals>.decorate_context(*args, **kwargs)
     24 @functools.wraps(func)
     25 def decorate_context(*args, **kwargs):
     26     with self.clone():
---> 27         return func(*args, **kwargs)

File ~/.local/lib/python3.9/site-packages/whisper/decoding.py:811, in decode(model, mel, options, **kwargs)
    808 if kwargs:
    809     options = replace(options, **kwargs)
--> 811 result = DecodingTask(model, options).run(mel)
    813 return result[0] if single else result

File ~/.local/lib/python3.9/site-packages/whisper/decoding.py:522, in DecodingTask.__init__(self, model, options)
    520 self.initial_tokens: Tuple[int] = self._get_initial_tokens()
    521 self.sample_begin: int = len(self.initial_tokens)
--> 522 self.sot_index: int = self.initial_tokens.index(tokenizer.sot)
    524 # inference: implements the forward pass through the decoder, including kv caching
    525 self.inference = PyTorchInference(model, len(self.initial_tokens))

ValueError: tuple.index(x): x not in tuple

```
Has anybody faced a similar issue? Any help would be appreciated. Thank you.

**Comment by OS-Chris** [comment]
I had the same error in decoding.py, but on a different line (still setting self.sot_index though). If you compare the line of the error your getting with the current source code, you'll see the source code has since been updated.

I simply updated my version of whisper to get it working.
`pip install -U openai-whisper`

**Comment by Suma-Rajashankar** [comment]
Thank you so much Chris for your feedback. Yes, I uninstalled openai-whisper and reinstalled it with the latest version and it's working now.
